pages/docs/1_query/advanced/why_provenance.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Tracking source attribution of query results'
+description: 'Comunica can annotate query result bindings with the sources used to produced these bindings.'
+---
+
+Comunica can track the sources that contribute to query results. This is equivalent to why-provenance of results.
+
+To achieve this, Comunica wraps all query sources in a `QuerySourceAddSourceAttribution` via `ActorQuerySourceIdentifyHypermediaAnnotateSource`. This actor sets the URL of each query source in the context of the bindings it produces.
```

Comment:
Maybe precede this by saying that source attribution is not enabled by default, and that it requires making a custom config. Maybe also link to one of the tutorials on making a custom config.